test/00_init.ts

```diff
@@ -25,7 +25,7 @@ const prefixes: Dictionary<true> = {};
 
 Bluebird.resolve(fs.promises.readdir(__dirname))
 	.call('sort')
-	.each((fileName) => {
+	.each(async (fileName) => {
 		const ext = path.extname(fileName);
 		if (ext !== '.ts') {
```

Comment:
Should this now become v ?
```suggestion
		if (ext !== '.ts' && ext !== '.js') {
```

Reply:
There's no need for it, it'd only be if we started writing tests in js directly or if we pre-compile the test files. Either way neither of those are part of this PR